sample-mail/src/main/java/com/hannesdorfmann/mosby3/sample/mail/MailApplication.java

```diff
@@ -18,6 +18,11 @@
 
   @Override public void onCreate() {
     super.onCreate();
+    if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
Arrrr ... have overlooked that you have already added it!
Forget about my last comment!

Thanks a lot!